dwave/cloud/config/models.py

```diff
@@ -23,7 +23,7 @@
 
 from dwave.cloud.config.loaders import update_config
 from dwave.cloud.api.constants import (
-    DEFAULT_REGION, DEFAULT_METADATA_API_ENDPOINT)
+    DEFAULT_REGION, DEFAULT_METADATA_API_ENDPOINT, DEFAULT_LEAP_API_ENDPOINT)
```

Comment:
Might be time to alphabetize the order of these imports

Reply:
Not yet. :rofl: